parquet-generator/src/main/java/org/apache/parquet/encoding/bitpacking/ByteBasedBitPackingGenerator.java

```diff
@@ -256,4 +254,8 @@ private static String align(int value, int digits) {
     result.append(valueString);
     return result.toString();
   }
+  
+  private static int paddedByteCountFromBits(int bitLength) {
```

Comment:
yeah sort of a bummer but not too bad. Can you add a comment to this method: 
```
// duplicated from BytesUtils to avoid a circular dependency between parquet-common and parquet-generator
// TODO: fix the circular dependency by moving the shared bits to a 3rd module (PARQUET-XXXX)
```